src/UI/Factory.php

```diff
@@ -336,4 +337,62 @@ public function panel();
 	 **/
 	public function modal();
 
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *     Popovers can be used if space is scarce to display secondary information such as a preview of
+	 *     some item or to offer limited user interaction.
+	 *   composition: >
+	 *     Popovers consist of a layer displayed above all other content.
+	 *     The content of the Popover depends on the functionality it performs.
+	 *     A Popover MAY display a title above its content.
+	 *     All Popovers contain a pointer pointing from the Popover to the Triggerer of the Popover.
```

Comment:
We can change it here but we would leave the interface since this should be made with another PR and developers should decide. For the documentation I can easily switch to Trigger if needed. Any opinions, @Amstutz, @alex40724 or @klees? If there is no feedback I will change it to Trigger in documentation. Maybe it will be a bit harder to understand for developers then, but for consumers of the rules it's maybe better.